Translators/WZDx/tim_generator.py

```diff
@@ -175,15 +179,35 @@ def get_first_road_name(feature):
 
 
 def get_data_frames(feature):
-    coords = feature["geometry"]["coordinates"]
+    coords = copy.deepcopy(feature["geometry"]["coordinates"])
     anchor = get_anchor(feature)
     if anchor is None:
         return None
     start_date = feature["properties"]["start_date"]
     duration = get_duration_time_minutes(feature)
     msgId = get_msg_id(anchor)
-    regions = [get_region(
-        coords, anchor, roadName=get_first_road_name(feature))]
+
+    # In here need to calculate if size of path is greater than 63

```

Comment:
Let's add in a unit test to show this new functionality

Reply:
Added unit test to test_tim_generator